framework/src/modules/http_api/controllers/accounts.js

```diff
@@ -27,42 +27,13 @@ function AccountsController(scope) {
 }
 
 function accountFormatter(totalSupply, account) {
-	const formattedAccount = _.pick(account, [
-		'address',
-		'publicKey',
-		'balance',
-		'nonce',
-		'asset',
-		'keys',
-		'votes',
-		'totalVotesReceived',
-		'unlocking',
-		'delegate',
-		'isDelegate',
-		'username',
-	]);
-
-	if (account.isDelegate) {
-		formattedAccount.delegate = _.pick(account, [
-			'fees',
-			'rewards',
-			'rewards',
-			'producedBlocks',
-			'missedBlocks',
-			'productivity',
-		]);
-
-		// Computed fields
-		formattedAccount.delegate.approval = calculateApproval(
-			formattedAccount.totalVotesReceived,
-			totalSupply,
-		);
-	}
-
-	// If an account is not multi signature, then do not include keys for response
-	if (formattedAccount.keys.numberOfSignatures === 0) {
-		delete formattedAccount.keys;
-	}
+	const formattedAccount = {
+		...account,
+		delegate: {
```

Comment:
Why not just add the approval to the original `delegate` property as the object is no longer used so mutating it does not affect anything else?